scripts/sanity_chk/sanity_compare.args

```diff
@@ -8,11 +8,3 @@ frdm_k64f
 nrf51_pca10028
 -p
 nrf52_pca10040
--p
-quark_d2000_crb
--p
-quark_se_c1000_devboard
--p
-quark_se_c1000_ss_devboard
--p
-altera_max10
```

Comment:
why was altera_max10 removed? It's a Nios II board

Reply:
altera_max10 is not being removed, here just cleaning up this more than I should have I guess.